pkgs/tools/backup/borgbackup/default.nix

```diff
@@ -15,41 +15,31 @@
 }:
 
 let
-  python = python3Packages.python.override {
-    packageOverrides = self: super: {
-      msgpack = super.msgpack.overrideAttrs (oldAttrs: rec {
-        version ="1.0.4";
-
-        src = fetchPypi {
-          pname = "msgpack";
-          inherit version;
-          hash = "sha256-9dhpwY8DAgLrQS8Iso0q/upVPWYTruieIA16yn7wH18=";
-        };
-      });
-    };
-  };
+  python = python3;
 in
 python.pkgs.buildPythonApplication rec {
   pname = "borgbackup";
-  version = "1.2.7";
-  format = "pyproject";
+  version = "1.2.8";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-9j8oozg8BBlxzsh7BhyjmoFbX9RF2ySqgXLKxBfZQRo=";
+    hash = "sha256-050isNLLdFWE1oYIoXm2x197QOSW6W/reJ5B00mR9Ko=";
   };
 
   postPatch = ''
     # sandbox does not support setuid/setgid/sticky bits
     substituteInPlace src/borg/testsuite/archiver.py \
-      --replace "0o4755" "0o0755"
+      --replace-fail "0o4755" "0o0755"
   '';
 
-  nativeBuildInputs = with python.pkgs; [
+  build-system = with python.pkgs; [
```

Comment:
Can this hunk be backported?

Reply:
The nixpkgs manual only mentions "build-system" in the unstable version.

Reply:
> Can this hunk be backported?

No.

Reply:
Then I think it should be a separate PR, so that we can backport the version update to stable.

Reply:
I'll split it into two commits so we can backport one of them if that's desired.